admin/tabs/general/roles/graph/RoleGraph.ts

```diff
@@ -27,22 +28,58 @@ export const roleGraph = hoistCmp.factory({
             compactHeader: true,
             icon: Icon.idBadge(),
             title: role ? `Relationships - ${role.name} ` : 'Relationships',
-            item: content(),
-            bbar: toolbar({
-                item: buttonGroupInput({
-                    bind: 'relationship',
-                    items: [
-                        button({
-                            value: 'inherited',
-                            text: `Inherited BY this Role (${role?.inheritedRoles.length})`
-                        }),
-                        button({
-                            value: 'effective',
-                            text: `Inheriting FROM this Role (${role?.effectiveRoles.length})`
-                        })
-                    ],
-                    width: 400
+            item: div({
+                item: div({
+                    style: {margin: 'auto'},
+                    item: content()
                 }),
+                style: {
+                    display: 'flex',
+                    flex: '1 0',
+                    overflow: 'auto'
+                }
+            }),
+            bbar: toolbar({
+                items: [
+                    buttonGroupInput({
+                        bind: 'relationship',
+                        items: [
+                            button({
+                                value: 'inherited',
+                                text: `Inheriting from ${role?.inheritedRoles.length} ${pluralize('role', role?.inheritedRoles.length)}`
+                            }),
+                            button({
+                                value: 'effective',
+                                text: `Granting to ${role?.effectiveRoles.length} ${pluralize('role', role?.effectiveRoles.length)}`
+                            })
+                        ]
+                    }),
+                    hspacer(10),
+                    buttonGroupInput({
+                        bind: 'inverted',
+                        items: [
+                            button({
+                                value: true,
+                                icon: Icon.treeList({rotation: 90})
+                            }),
+                            button({
+                                value: false,
+                                icon: Icon.treeList()
+                            })
+                        ]
+                    }),
+                    hspacer(10),
+                    'Width',
```

Comment:
Wonder if this label should be `Zoom`.  I just have a few roles in my local sandbox, so not a great test, but I found that adjusting the slider moved this little dot artifact around.

https://github.com/xh/hoist-react/assets/88904581/a9628f55-e741-4a2f-bd73-8f501a0817d6